samza-rest/src/main/java/org/apache/samza/monitor/LocalStoreMonitorConfig.java

```diff
@@ -46,7 +46,7 @@
    * Defines the comma separated list of job status servers of the form
    * "Host1:Port1,Host2:Port2".
    */
-  private static final String CONFIG_JOB_STATUS_SERVERS = "job.status.servers";
+  private static final String CONFIG_JOB_STATUS_SERVERS = "jobs.status.servers";
```

Comment:
Ahh didn't see those other properties in LocalStoreMonitorConfig, so from that perspective it makes sense. But... 

> How this collides with JobsResourceConfig?
> Why this is confusing?
 
Not collide, inconsistent. If a user configures samza-rest with both the jobs resource and this monitor they will have a mix of properties using "job." and "jobs." to refer to the same thing, right? 
`
job.proxy.factory.class=org.apache.samza.myClass
job.installations.path=./
jobs.status.servers=128.0.0.1
`
That will seem odd, and users will be unsure if there's some subtle difference they're missing.